coralproject/cay
# Edit submission

## What does this PR do?

This PR will allow you to edit a submission reply in the Gallery

## How do I test this PR?

- send a reply to the Gallery
- click Edit
- type some stuff into the box
- click Confirm and see the update displayed below the original text

@coralproject/frontend
